desktop/ui/src/main/java/org/datacleaner/windows/AnalysisJobBuilderWindowImpl.java

```diff
@@ -908,7 +908,7 @@ public void applyPropertyValues() {
     }
 
     private void onSourceColumnsChanged() {
-        boolean everythingEnabled = !_analysisJobBuilder.getSourceColumns().isEmpty();
+        boolean everythingEnabled = !_analysisJobBuilder.getSourceColumns().isEmpty() &&  _datastore != null;
```

Comment:
Now I'm totally being a pain in the behind, but there's a double space between `&&` and `_datastore`